### PR TITLE
Allow council to slash treasury tip

### PR DESCRIPTION
**Pull request by shamb0**
Polkadot Companion for [Allow council to slash treasury tip by shamb0 · Pull Request #7753 · paritytech/substrate](https://github.com/paritytech/substrate/pull/7753)

### CODE REVIEW

File: runtime/kusama/Cargo.toml [hunk 1]
```diff
@@ -61,7 +61,7 @@ pallet-staking-reward-curve = { package = "pallet-staking-reward-curve", git = "
 frame-system = {git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 frame-system-rpc-runtime-api = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
 pallet-timestamp = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
-pallet-tips = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
+pallet-tips = { git = "https://github.com/shamb0/substrate.git", branch = "issue-7057-allow-council-to-slash-treasury-tip", default-features = false }
```
Comment (shawntabrizi) [inline runtime/kusama/Cargo.toml:64]: Don't change this

File: runtime/kusama/src/weights/pallet_tips.rs [hunk 2]
```diff
@@ -35,43 +35,56 @@
 // --output=./frame/tips/src/weights.rs
 // --template=./.maintain/frame-weight-template.hbs
 
+
 #![allow(unused_parens)]
 #![allow(unused_imports)]
 
-use frame_support::{traits::Get, weights::Weight};
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
```
Comment (shawntabrizi) [inline runtime/kusama/src/weights/pallet_tips.rs:42]: Why is there a new import here?

File: runtime/kusama/Cargo.toml (hunk 1 above)
Comment (shamb0) [reply runtime/kusama/Cargo.toml]: Yes Shawn, got it & reverted it back.

File: runtime/kusama/src/weights/pallet_tips.rs (hunk 2 above)
Comment (shamb0) [reply runtime/kusama/src/weights/pallet_tips.rs]: Its my bad, removed it.

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
bot merge

**Pull request merged**